include/deal2lkit/fe_values_cache.h

```diff
@@ -23,7 +23,74 @@
 #include <deal2lkit/utilities.h>
 
 D2K_NAMESPACE_OPEN
-
+/**
+ * Helper class to simplify the assembly of non-linear problems, and the
+ * evaluation of finite element fields. This class will allow you to loop
+ * over cells, and access to temporary std::vectors of objects (of the same
```

Comment:
actually we don't loop over cells using this class, right? `This class allows you, while looping over cells, to access temporary etc.` ?